examples/web/src/App.tsx

```diff
@@ -1,6 +1,6 @@
 import CreativeEditorSDK from '@cesdk/cesdk-js';
+import { GetCutoutLibraryInsertEntry } from '@imgly/plugin-cutout-library-web';
```

Comment:
Any reason this is a upper case method name?

However, this is apparently only a workaround until we have a dock API published so that we can define the entries via API and not configuration.

Reply:
Changed